app/http/controllers/AuthController.ts

```diff
@@ -1,21 +1,24 @@
 import { NextFunction, Request, Response } from "express";
 import { AuthService } from "../../services/AuthService";
 
-
 export const AuthController = {
-	/**
-	 * Handles the login request
-	 * 
-	 * @param req {Request} Express request object
-	 * @param res {Response} Express response object
-	 * @param next {NextFunction} Express NextFunction (used for middleware)
-	 */
-	 async login(req: Request, res: Response, next: NextFunction) {
-		try {
-			const dat = await AuthService.validateLogin(req.body.username, req.body.password)
-			res.send(dat);
-		} catch (error) {
-			return next(error);
-		}
-	},
-}
\ No newline at end of file
+  /**
+   * Handles the login request
+   *
+   * @param req {Request} Express request object
+   * @param res {Response} Express response object
+   * @param next {NextFunction} Express NextFunction (used for middleware)
+   */
+
+  async Login(req: Request, res: Response, next: NextFunction) {
+    try {
+      const token = await AuthService.ldapJs(
+        req.body.username,
+        req.body.password
+      );
+      res.send(token);
```

Comment:
So I think this is sending to the client, just the plain text token.

Do we want to send the token in a JSON object? Actually asking, up to you guys